src/ai/backend/storage/server.py

```diff
@@ -31,6 +31,10 @@
 log = BraceStyleAdapter(logging.getLogger(__spec__.name))  # type: ignore[name-defined]
 
 
+async def health_check(request: web.Request) -> web.Response:
+    return web.json_response({"status": "OK"})
+
+
```

Comment:
API handlers on storage-proxy are defined under `src/ai/backend/storage/api/client.py` `src/ai/backend/storage/api/manager.py`, depending on its characteristic. Looks like the purpose of this health check API is for the client side so let's move it to client.py.

Reply:
This API is required from HAProxy to check the status of the Storage Proxy service, as stated in lablup/giftbox#198, so I think it is correct to implement  in `server.py`.

Reply:
On second thought, to query the status of Storage Proxy from the Control Panel, it is also required to implement in client.py ... Sorry for my confusion.

Reply:
Hm... I'm confused. Seems like implementing the status API in `server.py` would be enough to query it from Control Panel. What is needed in deploy the service via HAProxy is to return `200 OK` for `http://<storage-proxy-url>/status`.